certified-connectors/Dokobit Portal/apiDefinition.swagger.json

```diff
@@ -1384,6 +1382,147 @@
         ]
       }
     },
+    "/signing/{token}/update.json": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "Status",
+                  "x-ms-summary title": "Status"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "400",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "status",
+                  "x-ms-summary title": "Status"
+                },
+                "message": {
+                  "type": "string",
+                  "description": "message",
+                  "x-ms-summary title": "Message"
+                },
+                "errors": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "message": {
+                        "type": "string",
+                        "description": "message",
+                        "x-ms-summary title": "Message"
+                      },
+                      "error_code": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "error_code",
+                        "x-ms-summary title": "Error Code"
+                      },
+                      "field": {
+                        "type": "string",
+                        "description": "field"
+                      }
+                    }
+                  },
+                  "description": "errors"
+                },
+                "error_code": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "error_code",
+                  "x-ms-summary title": "Error Code"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Portal - Update Signing",
+        "description": "Allows to update attributes of an existing document signing.",
+        "operationId": "Portal-Update",
```

Comment:
@epelanis all operationIds should be in CamelCase. Please change to "PortalUpdate". please fix all operationIds in the swagger file.

Reply:
We used the same naming convention as in other existing operationIds. For example, "Portal-GetCategories". How come it was accepted previously?

Reply:
@Amjed-Ayoub We used the same naming convention as used in already existing "operationIds". For example, "Portal-GetCategories". How come those were accepted before?

Reply:
@Amjed-Ayoub We used the same naming convention as used in already existing "operationIds". For example, "Portal-GetCategories". How come those were accepted before?
Also, will changing the "operationId" have an effect on current users?

Reply:
Apologies for this but it looks like they were mistakenly approved. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

Reply:
@Amjed-Ayoub will the "operationId" changes have an effect on the existing flows of current users?

Reply:
@epelanis thank you so much for bringing this up. I think you are correct this will break existing flows of current users. I will double check I let you know, for now you can have them as they are. I will ignore this. 
Please resolve the below feedback. 
Thank you so much again.